src/containers/ResponsiveHeader.tsx

```diff
@@ -0,0 +1,175 @@
+import {
+  AppstoreFilled,
+  DatabaseFilled,
+  DingtalkSquareFilled,
+  FolderViewOutlined,
+  LogoutOutlined,
+  PlayCircleFilled,
+  SettingFilled,
+  StarFilled,
+  SwapOutlined,
+} from '@ant-design/icons';
+import { Divider, Input, Menu, Row } from 'antd';
+import { slide as BurgerMenu } from 'react-burger-menu';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import logo from '../assets/img/logo.png';

```

Comment:
logo is not found, has following error: Unable to resolve path to module '../assets/img/logo.png'